_includes/misc/secure-context.md

```diff
@@ -0,0 +1 @@
+A [secure context](https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts) on your web server is required. This generally means `https` must be used for any URL other than a local server. Modern browsers [restrict many new features to secure contexts](https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts/features_restricted_to_secure_contexts).
```

Comment:
I avoided saying `for real-time collaboration` because I wanted to make this generic and available to other features that require a secure context (such as proposed changes to powerpaste).

I'd also prefer to go with "requires" over "includes", since it's a _very_ strict requirement.

Reply:
I've had another crack at it, using some of your suggestions but keeping it generic. See what you think.